doc/DESCRIPTION.md

```diff
@@ -5,3 +5,5 @@ It can also create a MySQL database - which will be backed up and restored with
 PHP-FPM version can also be selected among `none`, `7.4`, `8.0`, `8.1` and `8.2`.
 
 **Once installed, go to the chosen URL to know the user, domain and port you will have to use for the SFTP access.** The password is one you chosen during the installation. Under the Web directory, you will see a `www` folder which contains the public files served by this app. You can put all the files of your custom Web application inside.
+
+The application can also lets you manage 404 errors - if you activate the post-install option. Simply create an`error` folder in the `www` root directory. 
```

Comment:
I'm confused ... it says "if you activate the post-install option" (which I understand as you meaning the config panel option)

But the install enables this by default ... ?

Imho we might want to make this opt-in and not opt-out .. in particular I'm worried about what happens if the mechanism is enabled but no 404.html exists ...

Reply:
Yes, I have active this option to test it quickly with my raspberry, I need to set it too false to have the legacy behavior.